docs/source/api/apollo-server.md

```diff
@@ -344,3 +344,7 @@ addMockFunctionsToSchema({
    itself. Set this to false to disable. You can manually invoke 'stop()' and
    'sendReport()' on other signals if you'd like. Note that 'sendReport()'
    does not run synchronously so it cannot work usefully in an 'exit' handler.
+
+*  `sendErrorTraces`: boolean
```

Comment:
Since this setting isn't really about whether or not to send a trace, maybe it should be something like `maskErrors` or `maskErrorDetails`?

Reply:
Totally! The name is definitely an artifact of evolving from not sending them completely to masking